src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -918,6 +918,25 @@ executing the TerminateProcess method of the process. Whereas the SIGKILL is equ
 the forceful removal of the running process.
 
 
+===== Termination Behavior by Language
+
+.Termination Behavior by Language
+|===
+|Language/Execution type|Bash |Python
+
+|Forked Mode
+| terminates immediately
+| receives SIGTERM and waits cleanup timeout
+
+
+|Non-Forked Mode
+| receives SIGTERM and waits cleanup timeout
+| terminates immediately
+
+
+|===
```

Comment:
I think you could say how to handle the signals for the various languages. (shutdown hook, trap, python trap)